serpent/src/consts.rs

```diff
@@ -1,25 +1,2 @@
 pub const PHI: u32 = 0x9e37_79b9;
 pub const ROUNDS: usize = 32;
```

Comment:
`consts.rs` is pretty bare at this point, I wasn't sure if it should be kept as is or just moved to `lib.rs`

Reply:
Yes, we can move these two constants to `lib.rs`.